client/src/components/form/Greeter.js

```diff
@@ -41,7 +41,7 @@ console.log(values)
             options={[
               { value: 'Aloha', label: 'Aloha' },
               { value: 'Bonjour', label: 'Bonjour' },
-              { value: 'Greetings and salutations', label: 'Greetings and salutations' },
+              { value: 'Greetings and salutations', label: 'o20ya1 - after changing secrets to vars - April 19, 2023' },
```

Comment:
Let's undo this change after now that you have tested since this is the generic sample app that everyone clones and isn't tied to a specific license plate

Reply:
Completely agree, this was an oversight on my side. I have just changed it.